J/jemalloc/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "jemalloc"
+version = v"5.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/jemalloc/jemalloc.git", "886e40bb339ec1358a5ff2a52fdb782ca66461cb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd jemalloc/
+autoconf
+
+if [[ "${target}" == "x86_64-linux-gnu" ]]; then
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls
+else
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+fi
+
+make
+make install
+
+# Rename .dll for Windows targets.
+if [[ "${target}" == *"w64"* ]]; then
+    mkdir -p ${libdir}
+    mv ${prefix}/lib/jemalloc.dll ${libdir}/libjemalloc.dll
+elif [[ "${target}" == "x86_64-linux-gnu" ]]; then
+    mv ${libdir}/libjemalloc.so.2 ${libdir}/libjemalloc.so
```

Comment:
Uh?

Reply:
`x86_64-linux-gnu` build was failing and then I added this and it seemed to pass...

Reply:
_How_ it was failing?

Reply:
Think you are right that this is not actually necessary. Rerunning a build.